group_vars/hypervisors.yml

```diff
@@ -14,4 +14,5 @@ neutron_ovs_bridge_mappings: "external:br-pf9"
 neutron_ovs_enable_tunneling: "False"
 neutron_ovs_net_type: vlan
 ceilometer_kvm_instance_disk_path: /opt/pf9/data/instances/
-
+ceilometer_customize: "False"
+ceilometer_cpu_interval: 300
```

Comment:
let's set this to the PF9 default of 600

Reply:
done